notion.py

```diff
@@ -2,43 +2,43 @@
 import os
 from datetime import datetime
 
+class NotionExporter:
```

Comment:
I'm modelling all exporters based on Readwise. They hold the necessary data (or initialized from command.py) instead of fetching it from an environment variable.

(Although it is not implemented)

Only Readwise is done as of now.